### ISSUE TITLE
Add dialog to confirm unit merge.

### ISSUE THREAD

**Issue by TrebuhD** [yiotro/Antiyoy#1]
Enhancement proposal. Add a dialog that says "are you sure you want to merge these 2 units"? It can be annoying when you accidentaly merge your units by touching the wrong tile. More problematic on small screens.

**Comment by yiotro** [comment]
I added this to plans, but I have to say that for me it's not really a problem. I have a smartphone with 3.8 inch screen, which is small enough.
1. If you accidentally merge wrong units you can just undo that easily.
2. On small screens it's better to zoom in.

**Comment by Techcable** [comment]
Agreed, you can easily undi it, and you should be zoomed in anyways.
